tests/kernel_performance/install_qatestset.pm

```diff
@@ -83,6 +83,12 @@ sub setup_environment {
         );
         assert_script_run("cat /root/qaset/qaset-setup.log");
     }
+
+    # Fill $ver_cfg by default value if it is undefined
+    unless ($ver_cfg) {
+        my $mybuild = check_var('BUILD', 'GM') ? "GM" : "Build" . get_var("BUILD", '');
+        $ver_cfg = "PRODUCT_RELEASE=SLES-" . get_var('VERSION') . ";PRODUCT_BUILD=$mybuild";
+    }
```

Comment:
Why not put this directly above around L52?
```
my $mybuild = check_var('BUILD', 'GM') ? "GM" : "Build" . get_var("BUILD", '');
my $ver_cfg = "PRODUCT_RELEASE=SLES-" . get_required_var('VERSION') . ";PRODUCT_BUILD=$mybuild";
$ver_cfg = get_var('VER_CFG', $ver_cfg);
```

Reply:
fixed, thank you!